### PR TITLE
BookA-Pset_GeophysicalSurveyCommon

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_GeophysicalSurveyCommon

## Description: 

Common properties for geophysical surveys

| Property | Datatype | Description |
|----------|----------|-------------|
|SurveyMethod|PEnum_GeophysicalSurveyMethod|Method of gephysical survey used|
|SurveyInstrumentManufacturer|IfcLabel|Manufacturer of the instrument used in the geophysical survey|
|SurveyInstrumentModel|IfcLabel|Model/name of the instrument used in the geophysical survey|
|SurveyDimension|IfcInteger|Dimension of the geophysical survey|

## Enumeration: PEnum_GeophysicalSurveyMethod

| Constant | Description |
|----------|-------------|
|REFLECTION_SEISMIC|Using seismic waves with near-vertical paths to detect impedance contrasts within the crust and upper mantle. |
|REFRACTION_SEISMIC|Utilizing the refraction of seismic waves on geology layers and rock/soil units to characterize subsurface geologic conditions. |
|SEISMIC|A  form of geophysical survey that aims at measuring the earth’s (geo-) properties by means of physical (-physics) principles such as magnetic, electric, gravitational, thermal, and elastic theories. It is based on the theory of elasticity and therefore tries to deduce elastic properties of materials by measuring their response to elastic disturbances called seismic (or elastic) waves. |
|SURFACEWAVE_SEISMIC|A non-disruptive, cost-effective way to investigate the shallow subsurface. The technique can be used to map the shear wave velocity and determine the depth to bedrock. |
|OTHER_SEISMIC|Any other type of seismic survey method. |
|MICROTREMOR|The microtremor survey method (MSM) is a non-invasive geophysical method used for geothermal exploration, which also uses ambient noise as a signal. An observation array is typically adopted to collect field data. |
|ELECTRICAL| Detecting the surface effects produced by electric current flow in the ground.  Using electrical methods, one may measure potentials, currents, and electromagnetic fields that occur naturally or are introduced artificially in the ground. The variation in electrical resistivity found in different rocks and minerals makes these techniques possible.|
|ELECTROMAGNETIC|Detecting the electrical properties of the subsurface by inducing electromagnetic energy within the subsurface and measuring the response of earth materials.|
|RADAR|Using radar pulses to image the subsurface. |
|GRAVITY|Measuring acceleration due to gravity at different points. |
|MAGNETIC|A type of geophysical survey relying on the contrast in magnetic properties between certain types of artifacts and their surroundings.|
|NMRS|A quantitative geophysical method that can be used to make in situ assessments of porosity, water content, mobile and immobile water fraction, and estimates of permeability. |

### CODE REVIEW

**Comment by SergejMuhic** [comment]
Constants missing descriptions.

File: IFC4x3/PropertyConstants/n/NMRS_0ch1vIyh55OB6vVQKSGdig.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NMRS_0ch1vIyh55OB6vVQKSGdig" Name="NMRS" UniqueId="26ac1e52-f2b1-4560-b1b9-7da51c427b2a">
+	<Documentation>A quantitative geophysical method that can be used to make in situ assessments of porosity, water content, mobile and immobile water fraction, and estimates of permeability.</Documentation>
```
Comment (larswik) [inline IFC4x3/PropertyConstants/n/NMRS_0ch1vIyh55OB6vVQKSGdig.xml]:
```suggestion
	<Documentation>Muclear Magnetic Resonance. A quantitative geophysical method that can be used to make in situ assessments of porosity, water content, mobile and immobile water fraction, and estimates of permeability.</Documentation>
```

File: IFC4x3/PropertyConstants/r/REFRACTION_SEISMIC_0KekqFNtX29QLcxE59er9U.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="REFRACTION_SEISMIC_0KekqFNtX29QLcxE59er9U" Name="REFRACTION_SEISMIC" UniqueId="14a2ed0f-5f78-4225-a566-ece149a3525e">
+	<Documentation>Utilizing the refraction of seismic waves on geology layers and rock/soil units to characterize subsurface geologic conditions</Documentation>
```
Comment (larswik) [inline IFC4x3/PropertyConstants/r/REFRACTION_SEISMIC_0KekqFNtX29QLcxE59er9U.xml]:
```suggestion
	<Documentation>Utilizing the refraction of seismic waves on geology layers and rock/soil units to characterize subsurface geologic conditions.</Documentation>
```

**Comment by kazunorit** [comment]
Please find my edit of the description below. Note that I deleted a constant "SEISMIC" because it is redundant and changed "MICROTREMOR" to "PASSIVE_SEISMIC" and "NMRS" to "NMR".

|Constant|Description|
|---|---|
|REFLECTION_SEISMIC|Using seismic waves reflected from the interface between formations having different physical properties to map geological structures and/or stratigraphic features.|
|REFRACTION_SEISMIC|Using the refraction of seismic waves on geology layers and rock/soil units to characterize subsurface geologic conditions.|
|SURFACEWAVE_SEISMIC|Using seismic waves that travel along (or near to) the surface to assess the surface wave velocity in the subsurface.|
|PASSIVE_SEISMIC|Using listening capabilities only, that is, no seismic energy is added by the investigator.|
|OTHER_SEISMIC|Any other types of seismic survey methods.|
|ELECTRICAL|Introducing current into the ground and measuring the distribution of the resulting potential in the ground to map electrical properties in the subsurface.|
|ELECTROMAGNETIC|Detecting the electromagnetic properties of the subsurface by inducing low-frequency electromagnetic fields within the subsurface and measuring the response of earth’s materials.|
|RADAR|Using radar pulses to image the subsurface.|
|GRAVITY|Measuring variations in the gravitational field of the earth to detect the density changes of subsurface bodies.|
|MAGNETIC|Measuring variations in the magnetic field of the earth which are related to changes of structures or magnetic susceptibility in the subsurface.|
|NMR|Measuring magnetic resonance signals from proton-containing fluids (e.g. water, hydrocarbons) in the earth's subsurface.|

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/PropertySets/Pset_GeophysicalSurveyCommon/Documentation.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Common properties for geophysical surveys
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/PropertySets/Pset_GeophysicalSurveyCommon/Documentation.md]:
```suggestion
Common properties for geophysical surveys.
```

File: IFC4x3/PropertyConstants/e/ELECTROMAGNETIC_0kMACCknX8GA5gomue0FFe.xml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ELECTROMAGNETIC_0kMACCknX8GA5gomue0FFe" Name="ELECTROMAGNETIC" UniqueId="2e58a30c-bb18-4840-a16a-cb0e2800f3e8">
+	<Documentation>Detecting the electromagnetic properties of the subsurface by inducing low-frequency electromagnetic fields within the subsurface and measuring the response of earth’s materials.</Documentation>
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/e/ELECTROMAGNETIC_0kMACCknX8GA5gomue0FFe.xml]:
```suggestion
	<Documentation>Detecting the electromagnetic properties of the subsurface by inducing low-frequency electromagnetic fields within the subsurface and measuring the response of earth&apos;s materials.</Documentation>
```

File: IFC4x3/PropertyConstants/s/SEISMIC_0oU_OBaif9JPTsxfB9XeID.xml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SEISMIC_0oU_OBaif9JPTsxfB9XeID" Name="SEISMIC" UniqueId="327be60b-92ca-494d-9776-ee92c986848d">
+	<Documentation>A  form of geophysical survey that aims at measuring the earth’s (geo-) properties by means of physical (-physics) principles such as magnetic, electric, gravitational, thermal, and elastic theories. It is based on the theory of elasticity and therefore tries to deduce elastic properties of materials by measuring their response to elastic disturbances called seismic (or elastic) waves.</Documentation>
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/s/SEISMIC_0oU_OBaif9JPTsxfB9XeID.xml]:
```suggestion
	<Documentation>A  form of geophysical survey that aims at measuring the earth&apos;s (geo-) properties by means of physical (-physics) principles such as magnetic, electric, gravitational, thermal, and elastic theories. It is based on the theory of elasticity and therefore tries to deduce elastic properties of materials by measuring their response to elastic disturbances called seismic (or elastic) waves.</Documentation>
```

File: IFC4x3/Properties/s/SurveyInstrumentModel_3DqMDOkJT9PgLG27L3f5fD/Documentation.md [hunk 6]
```diff
@@ -0,0 +1 @@
+Model/name of the instrument used in the geophysical survey
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/SurveyInstrumentModel_3DqMDOkJT9PgLG27L3f5fD/Documentation.md]:
```suggestion
Model/name of the instrument used in the geophysical survey.
```

File: IFC4x3/Properties/s/SurveyInstrumentManufacturer_0rqYHwzg5EY9Q2cWqaXGxR/Documentation.md [hunk 7]
```diff
@@ -0,0 +1 @@
+Manufacturer of the instrument used in the geophysical survey
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/SurveyInstrumentManufacturer_0rqYHwzg5EY9Q2cWqaXGxR/Documentation.md]:
```suggestion
Manufacturer of the instrument used in the geophysical survey.
```

File: IFC4x3/Properties/s/SurveyDimension_3BKUMNltrF3A3l2i5qFCSe/Documentation.md [hunk 8]
```diff
@@ -0,0 +1 @@
+Dimension of the geophysical survey
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/SurveyDimension_3BKUMNltrF3A3l2i5qFCSe/Documentation.md]:
```suggestion
Dimension of the geophysical survey.
```

**Comment by SergejMuhic** [comment]
@larswik I did not re-serialized but there were some errors. I did manual amendments. Hope I got all.

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**